src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Abstractions/IOpenApiConfigurationOptions.cs

```diff
@@ -39,5 +39,10 @@ public interface IOpenApiConfigurationOptions
         /// Gets or sets the value indicating whether to force the HTTPS protocol or not.
         /// </summary>
         bool ForceHttps { get; set; }
+
+        /// <summary>
+        /// Gets or sets the list of <see cref="IDocumentFilter"/> instances.
+        /// </summary>
+        List<IDocumentFilter> DocumentFilters { get; set; }

```

Comment:
This introduced a breaking change for us when updating from v1.2.0 to v1.3.0

Reply:
Oh, that's right. A new property was introduced to the existing interface - was overlooked. Apologies for any inconvenience caused.